lib/commands/context.js

```diff
@@ -184,6 +192,32 @@ helpers.startChromedriverProxy = async function startChromedriverProxy (context)
       if (androidPackage && androidPackage.length > 0) {
         opts.chromeAndroidPackage = androidPackage[1];
       }
+      if (!opts.extractChromeAndroidPackageFromContextName) {
+        if (_.has(this.opts, 'enableWebviewDetailsCollection') && !this.opts.enableWebviewDetailsCollection) {
+          // When enableWebviewDetailsCollection capability is explictly disabled, try to identify
+          // chromeAndroidPackage based on contexts, known chrome variant packages and queryAppState result
+          // since webviewsMapping does not have info object
+          const contexts = webviewsMapping.map((wm) => wm.webviewName);
```

Comment:
it would be helpful to add a log message here, which advices to enable details collection cap